pkg/api/converterfromv20180930preview.go

```diff
@@ -67,7 +67,7 @@ func mergeFromResourcePurchasePlanv20180930preview(oc *v20180930preview.OpenShif
 	}
 }
 
-func mergeProperties(oc *v20180930preview.OpenShiftManagedCluster, cs *OpenShiftManagedCluster) error {
+func mergePropertiesv20180930preview(oc *v20180930preview.OpenShiftManagedCluster, cs *OpenShiftManagedCluster) error {
```

Comment:
this is all a mouthful.
1) I can't remember why the converter code has to live in pkg/api - is it in any way feasible to move it out in to the subpackages?
2) failing that, create a `type v20180930converter struct{}` and make these methods of it - also less awful